src/unstable/components/Form/templates/ObjectFieldTemplate.tsx

```diff
@@ -1,26 +1,65 @@
+import { Accordion, AccordionPanel } from 'grommet';
 import * as React from 'react';
 import { ObjectFieldTemplateProps } from 'react-jsonschema-form';
+import { Heading } from '../../../../';
 
 const ObjectFieldTemplate = (props: ObjectFieldTemplateProps) => {
-	const { TitleField, DescriptionField } = props;
-	return (
-		<fieldset>
-			{props.schema.title && (props.uiSchema['ui:title'] || props.title) && (
-				<TitleField
-					id={`${props.idSchema.$id}__title`}
-					title={props.title || props.uiSchema['ui:title']}
-					required={props.required}
-				/>
-			)}
+	const { DescriptionField, idSchema } = props;
+
+	// The `activeIndex` and `onActive` props are used to make the accordion
+	// a controlled input, allowing the accordion to default to being open
+	const [active, setActive] = React.useState(true);
+
+	const toggleActive = () => setActive(!active);
+
+	const title =
+		props.schema.title && (props.uiSchema['ui:title'] || props.title)
```

Comment:
I would rather have a "ui:collapsible" or a similar option, as we might not always want to have a collapsible form whenever there is a title.